src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModel.java

```diff
@@ -420,9 +420,15 @@ protected static boolean isAltAfterAssembly(final PileupElement element, final b
      * @param priorList priors within the current ActiveRegion
      * @return prior VCs representing the same variant position as call
      */
-    List<VariantContext> getMatchingPriors(final Locatable curPos, final VariantContext call, final List<VariantContext> priorList) {
+    private List<VariantContext> getMatchingPriors(final Locatable curPos, final VariantContext call, final List<VariantContext> priorList) {
         final int position = call != null ? call.getStart() : curPos.getStart();
-        return priorList.stream().filter(vc -> position == vc.getStart()).collect(Collectors.toList());
```

Comment:
Add a comment that this method is implemented with an old-school for loop instead of streaming for performance reasons (otherwise, someone will probably come along in 2 years and undo this).